src/shared_modules/content_manager/src/action.hpp

```diff
@@ -68,6 +68,10 @@ class Action final
             {
                 // Use while with condition variable to avoid spurious wakeups.
                 std::unique_lock<std::mutex> lock(m_mutex);
+
+                // Run action on start, independently of the interval time.
+                runAction(ActionID::SCHEDULED);
```

Comment:
Improve: Add the same info message as in the other call

```suggestion
                std::cout << "Action: Initiating scheduling action for " << m_topicName << std::endl;
                runAction(ActionID::SCHEDULED);
```